### PR TITLE
MTM-59434 Announcement about endpoint access changes in microservice SDK

### PR DESCRIPTION
**Pull request by tsvet**
This PR contains the changelog to MTM-59434 and announces access restrictions to the /loggers spring boot endpoint in the Microservices SDK only to certain roles.

### CODE REVIEW

File: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - value: component-Sv2buFZ5I
+    label: Microservice SDK
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-58316
+version: 1020.110.0 
+---
+
+Microservices SDK exposes certain spring boot actuator endpoints, including /loggers, and provides an initial security configuration regarding the exposed spring boot actuator endpoints.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md:18]:
```suggestion
The Microservice SDK exposes certain Spring boot actuator endpoints, including `/loggers`, and provides an initial security configuration regarding the exposed spring boot actuator endpoints. 
```

File: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    label: Microservice SDK
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-58316
+version: 1020.110.0 
+---
+
+Microservices SDK exposes certain spring boot actuator endpoints, including /loggers, and provides an initial security configuration regarding the exposed spring boot actuator endpoints. 
+Due to security considerations, the POST-access to the /loggers endpoint has been restricted to the roles TENANT_ADMIN, TENANT_MANAGEMENT_ADMIN, APPLICATION_MANAGEMENT_ADMIN and APPLICATION_MANAGEMENT_UPDATE
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md:19]:
```suggestion
For security reasons, the POST access to the `/loggers` endpoint has been restricted to the roles TENANT_ADMIN, TENANT_MANAGEMENT_ADMIN, APPLICATION_MANAGEMENT_ADMIN and APPLICATION_MANAGEMENT_UPDATE.
```

File: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+date: 
+title: Restricting Access to Spring boot actuator endpoints
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md:3]:
```suggestion
title: Restricting access to Spring boot actuator endpoints
```

File: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+date: 
+title: Restricting access to Spring boot actuator endpoints
+product_area: Application enablement & solutions
+change_type:
+  - value: change-inv-3bw8e
+    label: Announcement
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md:7]: Shouldn´t it better be "API change"?

Comment (BeateRixen) [reply content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md:7]: @mgrumann Could you please take a look?

Comment (tsvet) [reply content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md]: After a discussion with @mgrumann , we decided to indeed change it to API change, as you proposed.

File: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  - value: component-Sv2buFZ5I
+    label: Microservice SDK
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-58316
+version: 1020.111.0 
+---
+
+The Microservice SDK exposes certain Spring boot actuator endpoints, including `/loggers`, and provides an initial security configuration regarding the exposed spring boot actuator endpoints.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md:18]:
```suggestion
The Microservice SDK exposes certain Spring Boot actuator endpoints, including `/loggers`, and provides an initial security configuration regarding the exposed Spring Boot actuator endpoints. 
```

File: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+date: 
+title: Restricting access to Spring boot actuator endpoints
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md:3]:
```suggestion
title: Restricting access to Spring Boot actuator endpoints
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Should be approved by @mgrumann.

**Review by mgrumann — APPROVED** [review]

File: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md (hunk 4 above)
Comment (tsvet) [reply content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md]: As we have changed this point already, I am resolving the conversation to be able to merge.

**Pull request merged**